src/bindings/c/ov/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+project(InferenceEngine_C_API)
```

Comment:
InferenceEngine is a legacy word. 
`OpenVINOC`? `openvino_c`? or something else

Reply:
We don't have better name so only follow the legacy convention. Maybe we need get a good name rule.

Reply:
My main concern that we shouldn't use InferenceEngine/ngraph names in new API.